fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/Quantifier.java

```diff
@@ -495,4 +524,45 @@ protected boolean needsRebase(@Nonnull final AliasMap translationMap) {
     public <Q extends Quantifier> Q narrow(@Nonnull Class<Q> narrowedClass) {
         return narrowedClass.cast(this);
     }
+
+    @Nonnull
+    public Optional<List<? extends QuantifiedColumnValue>> getFlowedValues() {
+        return flowedValuesSupplier.get();
+    }
+
+    @Nonnull
+    protected abstract Optional<List<? extends QuantifiedColumnValue>> computeFlowedValues();
+
+    @Nonnull
+    protected Optional<List<? extends Value>> unifyResultValues() {
```

Comment:
I am a little bit hesitant to use the word "unify" in the context of matching in an ordinary / general sense.

Reply:
Yes. It's not what some other people think this means!

Reply:
changed to `resolveValuesRangedOver`